### DISCUSSION TITLE
Notes field for short url?

### DISCUSSION THREAD

**Discussion by poblabs** [YOURLS/YOURLS#3816] [General]
Is it possible to display a notes field for the short url? Or can I use the Title for that (assuming the title doesn't change after the short url is created?)

I've been playing with plugins and it seems like I may be overthinking it if I can use the title field for notes on what the url is/was for

**Comment by dgw** [comment]
By default the title shouldn't be updated automatically unless it's empty.

Plugins can change this behavior, of course, but you can see here where the "infos" (stats) page checks for empty-string first:

https://github.com/YOURLS/YOURLS/blob/0d59b83a8c0ee0061143b0c660096c1d816036eb/yourls-infos.php#L21-L25

Since you can specify a custom title when creating a shorturl, it would be _a little_ silly if that value got overwritten later. :)

**Reply by poblabs** [reply]
I had a feeling I was overthinking it. Thanks!

**Reply by matttrv** [reply]
@dgw Just have to say, you're to be applauded for having such kindness in your reply.  So often we see less-than-kind "READ THE MANUAL" type of replies from devs, maintainers, and contributors.  But your tone and kindness....even sort of self-deprecation ("it would be a _little_ silly if that value got overwritten later") are all very welcoming and easy going.  Thank you for that :)

**Reply by dgw** [reply]
@matttrv I do have my moments sometimes, but yes, I _try_ to keep it professional. Glad it's noticed and appreciated :)
